e2e/kind.sh

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC2034
 KUBECONFIG="kind.yaml"
 KIND_CLUSTER="kind-cluster-kilo"
 KIND_BINARY="${KIND_BINARY:-kind}"
 KUBECTL_BINARY="${KUBECTL_BINARY:-kubectl}"
 KILO_IMAGE="${KILO_IMAGE:-squat/kilo}"
 
 is_ready() {
-	for pod in $(${KUBECTL_BINARY} -n ${1} get pods -o name -l ${2}); do
-		${KUBECTL_BINARY} -n ${1} get $pod | tail -n 1 | grep -q  Running;
-		if [ $? -ne 0 ]; then
+	for pod in $(${KUBECTL_BINARY} -n "$1" get pods -o name -l "$2"); do
+		if ${KUBECTL_BINARY} -n "$1" get "$pod" | tail -n 1 | grep -q  Running; then
```

Comment:
The test fails because the logic is reversed
```suggestion
		if ! ${KUBECTL_BINARY} -n "$1" get "$pod" | tail -n 1 | grep -q  Running; then
```

Reply:
I already reversed the inside of the `if` block to account for this though. But there's still something wrong?

Reply:
I don't understand, the inside of the `if` block still returns 1. Anyways, the block function blocks now if all pods are running and unblocks when they are not running.

Reply:
Ah sorry, i confused this with the block below where i flipped things around

Reply:
I meant to do this here as well